examples/gnn_link.py

```diff
@@ -210,7 +210,7 @@ def test(src_loader: NeighborLoader, dst_loader: NeighborLoader) -> np.ndarray:
             f"Val metrics: {val_metrics}"
         )
 
-        if val_metrics[tune_metric] > best_val_metric:
```

Comment:
why?

Reply:
when `val_metrics[tune_metric]` is always 0, state_dict will never be updated and stays `None`, resulting in an error later.